### ISSUE TITLE
Waht is the shape of the attention score for Paged Attention?

### ISSUE THREAD

**Issue by ChuanhongLi** [vllm-project/vllm#2481]
Thanks for your excellent work! Recently, I have been learning the source code of the vLLM. But, I have some difficulties in understanding 
codes: (csrc/attention/attention_kernels.cu)
```
In the kernel--paged_attention_kernel, 
// Compute softmax.
const float inv_sum = __fdividef(1.f, exp_sum + 1e-6f);
for (int i = thread_idx; i < num_tokens; i += NUM_THREADS) {
    logits[i] *= inv_sum;
}
```
Questions: 
1) Is the logits[i] storing the attetion score of the the ith token? 
2) Besides, I also want to learn what is the shape of the attention score ? 

Thanks for your help!

**Comment by hmellor** [comment]
https://github.com/vllm-project/vllm/blob/c64cf38673780544087af5ad5d3baf879a29220b/vllm/attention/ops/paged_attn.py#L99-L102
